alembic/versions/0f5be86d0fa0_decouple_resource_and_device_entities.py

```diff
@@ -0,0 +1,196 @@
+"""decouple resource and device entities
```

Comment:
Since you are modifying just resource and device models, you should not run fresh migration for the entire database hence this migration file should not contains migration for the entire database and you should only have one migration file, work with the backend team to guide you through this,

Reply:
Please implement @Bonifase feedback

Reply:
Okay thank you, lemme work on this

Reply:
I have worked on the feedback